Sail/src/Sail/entities/systems/network/HostSendToSpectatorSystem.cpp

```diff
@@ -0,0 +1,127 @@
+#include "pch.h"
+
+#include "HostSendToSpectatorSystem.h"
+
+#include "Sail/entities/components/NetworkSenderComponent.h"
+#include "Sail/entities/components/OnlineOwnerComponent.h"
+#include "Sail/entities/components/LocalOwnerComponent.h"
+#include "Sail/entities/Entity.h"
+
+#include "Network/NWrapperSingleton.h"
+#include "Sail/../../libraries/cereal/archives/portable_binary.hpp"
+
+#include "../src/Network/NWrapperSingleton.h"
+
+HostSendToSpectatorSystem::HostSendToSpectatorSystem() {
+	registerComponent<NetworkSenderComponent>(true, true, true);
+	registerComponent<TransformComponent>(true, true, false);
+}
+
+HostSendToSpectatorSystem::~HostSendToSpectatorSystem() {
+}
+
+void HostSendToSpectatorSystem::init(Netcode::PlayerID playerID) {
+	m_playerID = playerID;
+}
+
+
+// The messages this function creates must match the format of messages created by and received by NetworkSenderSystem and NetworkReceiverSystem
+void HostSendToSpectatorSystem::sendEntityCreationPackage(TCP_CONNECTION_ID TCPidPlayer) const {
+	std::ostringstream dataString(std::ios::binary);
+	Netcode::OutArchive ar(dataString);
+
+	// -+-+-+-+-+-+-+-+ Per-frame sends to per-frame receives via components -+-+-+-+-+-+-+-+ 
```

Comment:
is this comment relevant here?

"// -+-+-+-+-+-+-+-+ Per-frame sends to per-frame receives via components -+-+-+-+-+-+-+-+ "